brut.apktool/apktool-lib/src/main/java/brut/androlib/res/data/ResTable.java

```diff
@@ -260,16 +260,10 @@ public void setSharedLibrary(boolean flag) {
     }
 
     public void setSparseResources(boolean flag) {
-        if (mApkInfo.sparseResources != flag) {
-            LOGGER.info("Sparsely packed resources detected.");
-        }
         mApkInfo.sparseResources = flag;
     }
 
     public void setCompactEntries(boolean flag) {
-        if (mApkInfo.compactEntries != flag) {
-            LOGGER.info("Compactly packed resource entries detected.");
-        }
         mApkInfo.compactEntries = flag;
```

Comment:
Didn't like these in the output?

Reply:
Nah, this is redundant in output as it's already recorded as flags in apktool.yml.
This info means nothing to the end user.